docs/modules/control/move_to_a_pose_control/move_to_a_pose_control.rst

```diff
@@ -5,6 +5,12 @@ This is a simulation of moving to a pose control
 
 .. image:: https://github.com/AtsushiSakai/PythonRoboticsGifs/raw/master/PathTracking/move_to_pose/animation.gif
 
+For a more advanced version, see move_to_pose_robot_class.py. The program 
+implemented Robot class where you can define your robots and simulate robots 
+with differnt specificaitons at the same time.
+
+.. image:: https://user-images.githubusercontent.com/93126501/145834505-a8df8311-5445-413f-a96f-00460d47991c.png
```

Comment:
If you want add a gif, please submit a PR to https://github.com/AtsushiSakai/PythonRoboticsGifs/

Reply:
Ok. I will try to upload one after finishing this PR.